resources/external_pretrained_models.yaml

```diff
@@ -16,11 +16,11 @@ data:
   is_calibration: false
 - type: dataset
   dataset_type: pretrainedModelPackage
-  run: HSC/external
+  run: pretrained_models/tac_cnn_comcam_2025-02-18
```

Comment:
The substitution of `HSC/external` with `pretrained_models` looks a bit fishy. What's the story here? The original dataset path didn't have it as a subtring... 😕

Reply:
Good catch.  ci_imsim handles the collections a bit differently and I didn't catch it.